examples/src/main/python/ml/imputer.py

```diff
@@ -0,0 +1,46 @@
+#
```

Comment:
Prefer filename `imputer_example.py` to be consistent with other Python examples for ML